global-files/.github/labels.yml

```diff
@@ -137,3 +138,21 @@
 - name: "x:type/docs"
   description: "Work on Documentation"
   color: "3c2d9f"
+
+# The following Exercism-wide labels are used to influence how much reputation
+# a contributor gets for an authored/merged/reviewed PR.
+# See https://github.com/exercism/docs/blob/main/product/reputation.md
+
+- name: "reputation/contributed_code/major"
+  description: "Increases the number of reputation points awarded by this PR"
+  color: "fcfa9f"
+
+- name: "reputation/contributed_code/minor"
+  description: "Decreases the number of reputation points awarded by this PR"
+  color: "fcfa9f"
```

Comment:
@iHiD Thoughts on the color to use? Should we visually differentiate the major/minor reputation labels?